charts/plex/values.yaml

```diff
@@ -78,7 +78,25 @@ service:
   ## set the LoadBalancer service type to internal only.
   ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
   ##
-  annotations: {}
+  annotations:
+    metallb.universe.tf/allow-shared-ip: plex-svc
```

Comment:
I personally like the default annotation to help with metalLB; but do you think it's too opinionated to include that as the default?  I wonder if it should be shown as a commented example, similar to how we do it with the ingress annotations?

Reply:
Sure that works for me!